sumo/plotting/optics_plotter.py

```diff
@@ -20,6 +20,9 @@
 )
 
 
+def ev_to_nm(energy_ev):  # convert energy in eV to wavelength in nm
+    return 1239.84193 / energy_ev
```

Comment:
Can you do this using the `scipy.constants` module (https://docs.scipy.org/doc/scipy/reference/constants.html) rather than a magic number.

Reply:
Done! Was gonna do this at first but didn't know if it was better not to introduce a new module import (but already a required package I guess)